example/android/app/src/main/java/com/example/reactnativekeycommand/MainActivity.java

```diff
@@ -1,11 +1,20 @@
 package com.example.reactnativekeycommand;
 
+import android.view.KeyEvent;
+import com.expensify.reactnativekeycommand.KeyCommandModule;
+
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
 import com.facebook.react.ReactRootView;
 
 public class MainActivity extends ReactActivity {
 
+  @Override
+  public boolean onKeyDown(int keyCode, KeyEvent event) {
+    KeyCommandModule.getInstance().onKeyDownEvent(keyCode, event);
+    return super.onKeyDown(keyCode, event);
+  }
+
```

Comment:
Does this mean that the app installing this lib will require these native changes as well?

Reply:
Correct, let me add this in to README

Reply:
Is there any other solution, the user does not have to touch the native code? it will become much easier for them.

Reply:
not yet. `onKeyDown` event is captured by the ReactActivity class therefore we should bubble events down by modifying the class.

Reply:
Ok. I am good with it.